gpib_server.py

```diff
@@ -176,10 +189,26 @@ def read(self, c, n_bytes=None):
         Otherwise, reads until the device stops sending.
         """
         instr = self.getDevice(c)
-        if n_bytes is None:
-            ans = instr.read_raw()
+        if instr.resource_name.startswith('USB'):
+            try:
+                if n_bytes is None:
+                    ans = instr.read()
+                else:
+                    ans = instr.read(n_bytes)
+            except:
```

Comment:
Seems dangerous to have a bare except here. Generally, I think it's preferable to do `except Exception:` so as not to catch things like `KeyboardInterrupt`